docs/pages/develop/testing.mdx

```diff
@@ -0,0 +1,319 @@
+
+You will also use the [`jest-expo`](https://github.com/expo/expo/tree/main/packages/jest-expo) library, which is a Jest preset and mocks the native part of the Expo SDK and handles most of the configuration.
+
+## Installation and configuration
+
+If you have created your project using the [default Expo template](/get-started/create-a-project/), you can skip this section. The `jest-expo` and other required dev dependencies are already installed and configured.
+
+<Collapsible summary={<>Manual installation instructions for <CODE>jest-expo</CODE></>}>
+
+If you have created your project using [different template](/more/create-expo/#--template), follow the instructions below to install and configure `jest-expo` in your project.
+
+<Step label="1">
+
+Install `jest-expo` and other required dev dependencies in your project. Run the following command from your project's root directory:
+
+<Terminal
+  cmd={[
+    '# For all other package managers',
+    '$ npx expo install -- --save-dev jest-expo jest @types/jest',
+    '',
+    '# For yarn',
+    '$ yarn add -D jest-expo jest @types/jest',
+  ]}
+/>
+
+> **Note:** If your project is not using TypeScript, you can skip installing `@types/jest`.
+
+</Step>
+
+<Step label="2">
+
+Open **package.json**, add a script for running tests, and add the preset for using the base configuration from `jest-expo`:
+
+```json package.json
+{
+  "scripts": {
+    "test": "jest --watchAll" /* @end */
+    /* @hide ... */
+  },
+  "jest": {
+    "preset": "jest-expo"
+  }
+}
+```
+
+</Step>
+
+</Collapsible>
+
+<Collapsible summary={<>Additional configuration for using <CODE>transformIgnorePatterns</CODE></>}>
+
+You can transpile node modules your project uses by configuring [`transformIgnorePatterns`](https://jestjs.io/docs/configuration#transformignorepatterns-arraystring) in your **package.json**. This property takes a regex pattern as its value:
+
+<Tabs>
+
+<Tab label="npm/Yarn">
+
+```json package.json
+"jest": {
+  "preset": "jest-expo",
+  /* @info */
+  "transformIgnorePatterns": [
+    "node_modules/(?!((jest-)?react-native|@react-native(-community)?)|expo(nent)?|@expo(nent)?/.*|@expo-google-fonts/.*|react-navigation|@react-navigation/.*|@unimodules/.*|unimodules|@sentry/react-native|native-base|react-native-svg)"
+  ]
+  /* @end */
+}
+```
+
+</Tab>
+
+<Tab label="pnpm">
+
+```json package.json
+"jest": {
+  "preset": "jest-expo",
+  /* @info */
+  "transformIgnorePatterns": [
+    "node_modules/(?!(?:.pnpm/)?((jest-)?react-native|@react-native(-community)?|expo(nent)?|@expo(nent)?/.*|@expo-google-fonts/.*|react-navigation|@react-navigation/.*|@unimodules/.*|unimodules|@sentry/react-native|native-base|react-native-svg))"
+  ]
+  /* @end */
+}
+```
+
+</Tab>
+
+</Tabs>
+
+Jest has various configuration options. The above configuration covers the majority of your needs. However, you can always add to this pattern list. For more details, see [Configuring Jest](https://jestjs.io/docs/configuration).
```

Comment:
```suggestion
Jest has many configuration option, but the above configuration should cover most of your needs. However, you can always add to this pattern list. For more details, see [Configuring Jest](https://jestjs.io/docs/configuration).
```

Reply:
I think splitting the sentence is okay here. we don't have to add pause and introduce contradiction. Also, "options" is the plural form that references those multiple configs that Jest provides.

Reply:
I've taken the simplification parts from your suggestions. Let me know if you still think it's not good.

Reply:
oops, I meant to make that plural!